api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3961,6 +3961,48 @@ public void saveOrder_shouldAllowARetrospectiveOrderToCloseAnOrderThatExpiredInT
 		encounterService.saveEncounter(e2);
 		assertThat(new SimpleDateFormat("yyyy-MM-dd").format(o1.getDateStopped()), is("2008-08-14"));
 	}
+
+	/**
+	 * @see OrderService#saveOrderGroup(org.openmrs.OrderGroup, OrderContext)
+	 */
+	@Test
+	public void saveOrderGroup_shouldSaveOrderGroupWithOrderContext() {
+		executeDataSet(ORDER_SET);
+		Encounter encounter = encounterService.getEncounter(3);
+		OrderSet orderSet = Context.getOrderSetService().getOrderSet(1);
+		OrderGroup orderGroup = new OrderGroup();
+		orderGroup.setOrderSet(orderSet);
+		orderGroup.setPatient(encounter.getPatient());
+		orderGroup.setEncounter(encounter);
+
+		Order firstOrder = new OrderBuilder().withAction(Order.Action.NEW).withPatient(1).withConcept(10).withOrderer(1)
+			.withEncounter(3).withDateActivated(new Date()).withOrderType(17)
+			.withUrgency(Order.Urgency.ON_SCHEDULED_DATE).withScheduledDate(new Date()).build();
+
+		Order secondOrder = new OrderBuilder().withAction(Order.Action.NEW).withPatient(7).withConcept(10).withOrderer(1)
+			.withEncounter(3).withDateActivated(new Date()).withOrderType(17)
+			.withUrgency(Order.Urgency.ON_SCHEDULED_DATE).withScheduledDate(new Date()).build();
+
+		orderGroup.addOrder(firstOrder);
+		orderGroup.addOrder(secondOrder);
+
+		OrderType orderType = new OrderType();
+		orderGroup.setId(1);
+		orderType.setOrderTypeId(17);
```

Comment:
There are over a dozen order types in the standard test dataset.  Just get an existing one here and use it.